Entity/OverrideLeadRepository.php

```diff
@@ -344,6 +344,117 @@ protected function parseExtendedSearchFilters($parseFilters, $qb, $expressions,
         }
     }
 
+    /**
+     * Adds the "catch all" where clause to the QueryBuilder.
+     *
+     * @param \Doctrine\ORM\QueryBuilder|\Doctrine\DBAL\Query\QueryBuilder $q
+     * @param                                                              $filter
+     *
+     * @return array
+     */
+    protected function addCatchAllWhereClause($q, $filter)
+    {
+        $filterContainsPhone = $this->filterContainsPhone($filter); // might need to add a '+' so pass by reference
```

Comment:
This variables are only used once, you could just pass them into line 361:
`
$columns = array_merge(
    $this->filterContainsPhone($filter),
    // ...
);
`

Reply:
even if the vars are only getting referenced once (after assignment), it is a little cleaner to read IMO 🙂

Reply:
Let's keep it separate. You could add $columns to each of these methods and pass the array by reference and avoid an array merge all together. But as it stands it's fine I think, easy to read. We can always refactor when we hit this again.